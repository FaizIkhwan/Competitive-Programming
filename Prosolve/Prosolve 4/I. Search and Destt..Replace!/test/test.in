Yâ€™all have heard about search and destroy, right?
5
? !
a 4
i l
o @
h $