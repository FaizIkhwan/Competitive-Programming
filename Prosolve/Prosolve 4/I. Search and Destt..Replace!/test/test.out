Yâ€™4ll $4ve $e4rd 4b@ut se4rc$ 4nd destr@y, rlg$t!
